dom/rx-typings.d.ts

```diff
@@ -1,7 +0,0 @@
-import {EventsFnOptions} from './lib/DOMSource';
-import {Observable} from 'rx';
-export interface DOMSource {
-  select(selector: string): DOMSource;
-  elements(): Observable<Element>;
-  events(eventType: string, options?: EventsFnOptions): Observable<Event>;
-}
```

Comment:
Ah, I'll put them back. I was only checking if the build happened and tests passed, and from that perspective these seemed like clutter.